app/views/merchant_site/items/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Edit <%= @item.name %> </h1>
```

Comment:
Our form for updating an item. Could make it into a partial but it might be extra work